serverless.yml

```diff
@@ -10,6 +10,7 @@ custom:
     firebaseApp: ${file(secrets.yml):firebaseApp}
     travisToken: ${file(secrets.yml):travisToken}
     googleMaps: ${file(secrets.yml):googleMaps}
+    slackSlashCommandHook: ${file(secrets.yml):slackSlashCommandHook}
```

Comment:
I'm not familiar with the structure of this URL. I'm wondering if we could break it up into its components so that we could simplify this constant. We also need to update this file-

https://github.com/EsteemedIO/esteemed-api/blob/master/example.secrets.yml